tests/integration/targets/disk_info/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+####################################################################
+# WARNING: These are designed specifically for Ansible tests       #
+# and should not be used as examples of how to write Ansible roles #
+####################################################################
+
+# Test code for the disk_info module
+# Copyright 2021 VMware, Inc
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+- name: "Installing the psutil module"
+  pip:
+    name: psutil < 5.7.0
+    # Version 5.7.0 breaks on older pip versions. See https://github.com/ansible/ansible/pull/7066
```

Comment:
Can't older version of `psutil` be used for older pip versions and latest `psutil` for newer versions?
Also I can't see how the written PR is relevant to this issue.

Reply:
yes that seems generic. Should I check pip version and use specific version based on that @aminvakil ?

Reply:
Does it really depend on pip versions, or on Python versions? Also the URL you mention there seems wrong.

Reply:
If the problem is with pip version, yes.
Thanks for fixing this!

Reply:
@saranyasridharan Is this the PR you wanted to mention? https://github.com/ansible/ansible/pull/70667

Reply:
yes that seems to be the one. But that says merged. Doesn't that mean this error should not happen?

Reply:
No, `ansible-runner` integration test fails with `psutil > 5.7.0 `, so that PR is fixing `psutil` version to be less than 5.7.0.

Reply:
oh ok. My test also installs version less than 5.7.0 only. then ideally it shouldnt fail right?